test/ProductConstructionService.ScenarioTests/ScenarioTestBase.cs

```diff
@@ -979,10 +989,12 @@ protected static async Task WaitForMergedPullRequestAsync(string targetRepo, str
                 return;
             }
 
-            await Task.Delay(TimeSpan.FromSeconds(30));
+            await Task.Delay(WAIT_DELAY);
```

Comment:
This technically makes the check wait less, but not sure if it is that big of a difference considering other pull request merge checks have been waiting for 20 seconds in between anyways.

Can increase attempt count to compensate if anything

Reply:
I guess it should be fine, it should technically take 5-6 minutes - depending on how busy the service gets with the other subscriptions from other E2E tests. If this keeps failing, we can increase back up